config-example.yaml

```diff
@@ -137,6 +137,8 @@ db_path: ./db.sqlite
 
 # # Postgres config
 # If using a Unix socket to connect to Postgres, set the socket path in the 'host' field and leave 'port' blank.
+# Also if other 'sslmode' is required instead of 'require(true)' and 'disabled(false)', set the 'sslmode' you need
+# in the 'db_ssl' field. Refers to https://www.postgresql.org/docs/current/libpq-ssl.html Table 34.1.
```

Comment:
Can you move this line so it belongs to the `db_ssl:` entry?

Reply:
Sure I'll.